Formula/zork.rb

```diff
@@ -0,0 +1,30 @@
+class Zork < Formula
+  desc "The Dungeon modified from FORTRAN to C"
+  homepage "https://github.com/devshane/zork"
+  url "https://github.com/devshane/zork/archive/v1.0.0.tar.gz"
+  sha256 "ceedb436cf552bd8ca39abd3e85bae986c3f5df6f31c1bf2806d1b4c8509da67"
+  head "https://github.com/devshane/zork.git"
+
+  depends_on "cmake" => :build
```

Comment:
Hm, it looks like we aren't actually, just `$(CC)` so, I assume that's just `gcc`. Should I include that as a build dependency then?